### ISSUE TITLE
Kickstarter: Adding Authorization Server with "auth-server" in name break the API! dialog

### ISSUE THREAD

**Issue by thomsch98** [Haufe-Lexware/wicked.haufe.io#77]
To reproduce:

* setup your "usual" protal configuration
* add an api with oauth2 for authentication
* add a new authorization server named "a-auth-server"
* save auth server
* open API dialog

Output (on screen):

```
/usr/src/app/views/layout.jade:125 123| mixin checkBox(source, thisIsNotUsedButApparentlyJadeNeedsItSomebodyCanExplainPlease, label, desc) 124| - var inputId = source > 125| - var inputValue = eval(source) 126| .checkbox 127| label 128| if inputValue server is not defined

ReferenceError: /usr/src/app/views/layout.jade:125
    123| mixin checkBox(source, thisIsNotUsedButApparentlyJadeNeedsItSomebodyCanExplainPlease, label, desc)
    124|   - var inputId = source
  > 125|   - var inputValue = eval(source)
    126|   .checkbox
    127|     label
    128|       if inputValue

server is not defined
    at eval (eval at jade_interp (eval at <anonymous> (/usr/src/app/node_modules/jade/lib/index.js:218:8)), <anonymous>:1:28)
    at Object.jade_interp [as checkBox] (eval at <anonymous> (/usr/src/app/node_modules/jade/lib/index.js:218:8), <anonymous>:1281:18)
    at Object.eval (eval at <anonymous> (/usr/src/app/node_modules/jade/lib/index.js:218:8), <anonymous>:302:24)
    at Object.jade_interp [as renderApi] (eval at <anonymous> (/usr/src/app/node_modules/jade/lib/index.js:218:8), <anonymous>:320:4)
    at eval (eval at <anonymous> (/usr/src/app/node_modules/jade/lib/index.js:218:8), <anonymous>:2077:25)
    at eval (eval at <anonymous> (/usr/src/app/node_modules/jade/lib/index.js:218:8), <anonymous>:2095:4)
    at eval (eval at <anonymous> (/usr/src/app/node_modules/jade/lib/index.js:218:8), <anonymous>:2306:22)
    at res (/usr/src/app/node_modules/jade/lib/index.js:219:38)
    at Object.exports.renderFile (/usr/src/app/node_modules/jade/lib/index.js:380:38)
    at Object.exports.renderFile (/usr/src/app/node_modules/jade/lib/index.js:370:21)
```

**Comment by DonMartin76** [comment]
OK, so embarrassing enough, this has nothing to do with the term `a-auth-server`, and everything with the fact that it contains **two** hyphens, and the guard which replaces `-` with `_` was not implemented as `.replace(/\-/g, '_')` but as `.replace('-', '_')`...

**Comment by DonMartin76** [comment]
Fixed in 0.11.7.
